doc/source/release-notes/Percona-Server-5.7.10-3.rst

```diff
@@ -0,0 +1,41 @@
+.. rn:: 5.7.10-3
+
+===========================
+ |Percona Server| 5.7.10-3
+===========================
+
+Percona is glad to announce the first GA (Generally Available) release of |Percona Server| 5.7.10-3 on February 16th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.10-3/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.7.10 <http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-10.html>`_, including all the bug fixes in it, |Percona Server| 5.7.10-3 is the current Release Candidate release in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.7.10-3 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.7.10-3>`_
+
+New Features
+============
+
+ Complete list of changes between |Percona Server| 5.6 and 5.7 can be seen in :ref:`changed_in_57`.
+
+ |Percona Server| has implemented :ref:`Multi-threaded asynchronous LRU flusher <mt_lru_flusher>`.
```

Comment:
This work also allows to safely use "backoff" value for the innodb_empty_free_list_algorithm server system variable, and its default has been changed accordingly.